source/torque_vector/main.c

```diff
@@ -94,16 +94,12 @@ SPI_InitConfig_t spi_config = {
     .nss_sw = true,
     .nss_gpio_port = SPI_CS_MAG_GPIO_Port,
```

Comment:
if we are getting rid of bmm stuff, should this port be one of the accel or gps ports?